src/actions/myActivitiesActions.js

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+
```

Comment:
Does a white space in between the imports of essence here ? 😟

Reply:
Separation of external imports and custom imports